py/selenium/webdriver/remote/webdriver.py

```diff
@@ -888,6 +888,17 @@ def minimize_window(self):
         """
         self.execute(Command.MINIMIZE_WINDOW)
 
+    def print_page(self, print_option_arg = None):
+        """
+        Takes PDF of the current page.
+        The driver makes a best effort to return a PDF based on the provided parameters.
+        """
+        options = {}
```

Comment:
We don't need another dict here, we can just use populate `print_option` with the `to_dict()` method

Reply:
`print_options` are kept to be optional, the extra `options` is to make sure that I don't call `to_dict` on `None`.